### PR TITLE
keystone/issue#12-makeAPIstatic

### PR DESCRIPTION
**Pull request by cxoltero**
kenzanlabs/keystone#7
kenzanlabs/keystone/pipeline-validate-css #12

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
Can one of the admins verify this patch?

File: test/index.spec.js [hunk 1]
```diff
@@ -44,30 +44,30 @@ describe('check inner functions spies', function() {
...
     spy.should.have.been.calledWith('Restoring CSS Filter.');
   });
 
- //  it('Invalid file simulation', function () {
- //    var stream = function(){
- //    	gulp
-	//     .src(fixtures('test-css1.css'))
-	//     .pipe(validatePipeline().validateCSS());
-	//   };
+  //  it('Invalid file simulation', function () {
```
Comment (obuckley-kenzan) [inline test/index.spec.js]: Can we add a `TODO` here with why this is commented out and what the appropriate next steps are to enable it (ideally in an issue)?

**Comment by obuckley-kenzan** [comment]
@cxoltero I think the README should be updated too, yes?

File: test/index.spec.js (hunk 1 above)
Comment (paul-barry-kenzan) [inline test/index.spec.js]: If the tests are valid, I prefer to not comment out, but use `it()` without a callback. This disables/skips the tests, and makes devs aware that some tests aren't being run. A `@TODO` is still warranted, I just feel the commented specs are clutter. 

For example: 
```javascript
// @TODO Write tests to handle invalid files
it('should throw when an invalid file is provided'); // or similar
```

**Comment by paul-barry-kenzan** [comment]
@cxoltero @obuckley-kenzan 
Aside from the commented test comments, this looks great!

File: src/index.js [hunk 2]
```diff
@@ -7,42 +7,42 @@ var gutil = require('gulp-util');
...
+
+module.exports = {
+  validateCSS: function() {
+    return pipelineFactory();
   }
+};
+
+
+function pipelineFactory() {
+  var stream;
```
Comment (obuckley-kenzan) [inline src/index.js]: maybe it's how we're applying eslint, but in other pipeline we just do
```javascript
var stream = lazypipe() {
  //stuff
}
```
https://github.com/kenzanlabs/pipeline-minify-js/pull/23/files

**Comment by obuckley-kenzan** [comment]
ok to test

**Pull request merged**